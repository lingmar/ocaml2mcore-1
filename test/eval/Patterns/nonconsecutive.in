let foo x =
  match x with
    1 -> "1"
  | 2 -> "2"
  | 3 -> "3"
  | 10 -> "10"
  | 100 -> "100"
  | _ -> "Something else"

let _ =
  foo 1 |> print_endline;
  foo 2 |> print_endline;
  foo 3 |> print_endline;
  foo 4 |> print_endline;
  foo 10 |> print_endline;
  foo 100 |> print_endline;
  foo (-13) |> print_endline;
  foo 42 |> print_endline;
