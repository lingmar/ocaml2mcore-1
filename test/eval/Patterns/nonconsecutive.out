1
2
3
Something else
10
100
Something else
Something else